Src/Plugins/Tests.Security/ReflectionTests.cs

```diff
@@ -7,6 +7,8 @@
 using System.Linq;
 using System.Reflection;
 
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.SecurePlaintextSecretsValidators;
```

Comment:
>SecurePlaintextSecretsValidators [](http://example.com/codeflow?start=67&length=32)

Please make the file move without changing any using statement or namespace.

Thanks!

I understand you're trying to keep everything in line with directory structure but we didn't have a problem with our namespaces before, really, which aren't entirely consistent with current directory structure.

The problem is that the JSON files are hard to locate buried in dozens of validators, that was the complaint I remarked.

I really do appreciate that you tried to make a more assertive fix here, but when I see changes like this specific one that add a new 'using' with no discernable increase in value, I just think we're cluttering code unnecessarily.

Reply:
Ok!
Just updated!